src/Microsoft.ML.Data/Transforms/Normalizer.cs

```diff
@@ -67,37 +67,37 @@ public enum NormalizerMode
 
         public abstract class ColumnBase
         {
-            public readonly string Input;
-            public readonly string Output;
+            public readonly string Name;
+            public readonly string SourceColumnName;
             public readonly long MaxTrainingExamples;
 
-            private protected ColumnBase(string input, string output, long maxTrainingExamples)
+            private protected ColumnBase(string outputColumnName, string sourceColumnName, long maxTrainingExamples)
```

Comment:
>outputColumnName [](start = 48, length = 16)

so it should be `name` here, right? as you said in another comment.

Reply:
yep, just like line 70! thanks. 

---
In reply to: [251642026](https://github.com/dotnet/machinelearning/pull/2239#discussion_r251642026) [](ancestors = 251642026)